src/main/java/by/intexsoft/ignatieva/Printer.java

```diff
@@ -0,0 +1,11 @@
+package by.intexsoft.ignatieva;
+
+import java.util.Arrays;
+
+public class Printer {
+    public static void print(int[][] matrix) {
+        for (int i = 0; i < matrix.length; i++) {
+            System.out.println(Arrays.toString(matrix[i]));
```

Comment:
![image](https://user-images.githubusercontent.com/14859301/100108553-f3bc6f80-2e7b-11eb-9e69-309f10fa55b7.png)
lets improve printing for big matrix.